srtcore/epoll.cpp

```diff
@@ -125,10 +125,66 @@ ENOMEM: There was insufficient memory to create the kernel object.
    pair<map<int, CEPollDesc>::iterator, bool> res = m_mPolls.insert(make_pair(m_iIDSeed, CEPollDesc(m_iIDSeed, localid)));
    if (!res.second)  // Insertion failed (no memory?)
        throw CUDTException(MJ_SETUP, MN_NONE);
+   if (pout)
+       *pout = &res.first->second;
 
    return m_iIDSeed;
 }
 
+int CEPoll::clear_usocks(int eid)
```

Comment:
This allows to remove all subscriptions from eid, but leave eid ready for further use.